tests/test_workflow_api.py

```diff
@@ -11,6 +11,11 @@
 )
 
 
+def test_post_workflow_script_unauthorized(client, auth, workflow_mongo_coll, asset_workflow1):
```

Comment:
Do you need `auth` and `workflow_mongo_coll` fixture here?

Reply:
Good catch, not needed. I forgot to remove them when copied from the test case below for the workflow endpoint test.